.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,81 @@
+name: Docker Image CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      bin_tag:
+        description: "Okteto bin tag"
+        required: true
+        type: string
+
+jobs:
+  build_and_push:
+    name: Push to DockerHub
+    runs-on: "ubuntu-latest"
+
+    defaults:
+      run:
+        working-directory: .
+
+    steps:
+      - name: Checking out the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Context for Buildx
+        id: buildx-context
+        run: |
+          docker context create builders
+
+      - name: Set up Docker Builder
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+          install: true
+          endpoint: builders
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: arm64,amd64
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
```

Comment:
is there any reason to use `v1` here and `v2` here?
https://github.com/okteto/bin-image/pull/4/files#diff-4f9e38227ed64fefb17f4668a7ac4ab55b6149994d5ac2fd96182d5958479b54R56